crud/select/plan.lua

```diff
@@ -64,6 +66,11 @@ local function extract_sharding_key_from_scan_value(scan_value, scan_index, shar
 
     -- check that sharding key is included in the scan index fields
     local sharding_key = {}
+    local sharding_key_fieldno_map
+    local ddl_sharding_key = sharding.get_ddl_sharding_key(space_name, vshard.router.routeall())
```

Comment:
Plan is not good place to require vshard. If your need something you can pass it as argument.

Reply:
it is not there now